tests/unit/src/main/scala/tests/BaseLspSuite.scala

```diff
@@ -51,6 +53,24 @@ abstract class BaseLspSuite(suiteName: String) extends BaseSuite {
     sh.shutdown()
   }
 
+  override def munitTestTransforms: List[TestTransform] =
+    super.munitTestTransforms :+
+      new TestTransform(
+        "Print exception Stack",
+        { test =>
+          test.withBody(() =>
+            test
+              .body()
+              .andThen {
+                case Failure(exception) =>
+                  exception.printStackTrace()
```

Comment:
Without this, munit will not print the actual exception.

Reply:
The `-F` option should print the full original stack trace https://scalameta.org/munit/docs/troubleshooting.html#print-full-stack-traces

It might make sense to enable this by default in CI

Reply:
I would avoid using this workaround here because I think it will make all test failures very verbose

Reply:
ooo I didn't know about the `-F` option. I think that's a good idea to use for this.

Reply:
The MUnit docs are missing a reference table that documents all of the flag options. Most of the flags in the table here should work https://github.com/sbt/junit-interface 

The `-F` flag name is the same as for ScalaTest.

I regret using cryptic single-character flags instead of more descriptive names 😢

Reply:
The only time I've needed the `-F` flag was to debug an issue in MUnit internals. If the stack trace is hidden it indicates the MUnit framework is failing with an unexpected exception